python/prompt_chatgpt/src/main.py

```diff
@@ -0,0 +1,35 @@
+import openai
+from utils import get_static_file, throw_if_missing
+import os
+
+
```

Comment:
Double line break. also in utils.py

Reply:
This is the output of black formatter, I think we should keep it